### ISSUE TITLE
clipPath support

### ISSUE THREAD

**Issue by vorg** [runemadsen/rune.js#32]
Any ideas or plans for clipping / masking support in the future releases?

**Comment by runemadsen** [comment]
Yes, I would love to implement this. I'm also looking for contributors to help with the library :)
